.circleci/config.yml

```
@@ -97,6 +98,29 @@ commands:
           command: |
             curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
             sudo apt update && sudo apt install -y libssl1.1
+  make-artifact:
+    parameters:
+      target:
+        description: "Rust target to put in artifact"
+        type: string
+      suffix:
+        description: "Suffix that is on the binary"
+        type: string
+        default: ""
+    steps:
+      - run:
+          name: Make artifact
+          command: |
+            mkdir shuttle
+            mv target/<< parameters.target >>/release/cargo-shuttle<< parameters.suffix >> shuttle/shuttle<< parameters.suffix >>
+            mv LICENSE shuttle/
+            mv README.md shuttle/
+            mkdir artifacts
+            tar -cvzf artifacts/shuttle-${CIRCLE_TAG}-<< parameters.target >>.tar.gz shuttle
+      - persist_to_workspace:
+          root: artifacts
+          paths:
+            - shuttle-${CIRCLE_TAG}-<< parameters.target >>.tar.gz
```

For better support from tools like cargo-binstall, use {crate_name}-{version}-{target}.tar.gz
```suggestion
            - cargo-shuttle-${CIRCLE_TAG}-<< parameters.target >>.tar.gz
```